packages/react/src/components/TextInput/__tests__/PasswordInput-test.js

```diff
@@ -0,0 +1,352 @@
+/**
+ * Copyright IBM Corp. 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PasswordInput from '../PasswordInput';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+
+const prefix = 'cds';
+
+describe('PasswordInput', () => {
+  describe('renders as expected - Component API', () => {
+    it('should spread extra props onto input element', () => {
+      render(
+        <PasswordInput
+          data-testid="test-id"
+          id="input-1"
+          labelText="TextInput label"
+        />
+      );
+
+      expect(screen.getByTestId('test-id')).toHaveClass(
```

Comment:
If we're specifically looking for the input element, `getByLabelText` can be used instead of `getByTestId`.

```suggestion
      expect(screen.getByLabelText("TextInput label")).toHaveClass(
```